src/components/BadgeButtonList.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styled from "styled-components";
+import { content } from "../api/content";
+import BadgeButton from "./BadgeButton";
+
+const BadgeButtonSection = styled.div`
```

Comment:
Could it make sense to use a section instead of a div? Only thought about that cause of the name `BadgeButtonSection`. :)